careers/blocks/featured-card/featured-card.css

```diff
@@ -0,0 +1,121 @@
+.section.featured-card-container {
+  padding-top: var(--layout-size-xl);
+  padding-bottom: var(--layout-size-xl);
+}
+
+.featured-card-wrapper-no-background {
+  padding-bottom: var(--layout-size-xs) !important;
+  background-color: var(--static-color-black) !important;
```

Comment:
Is it not possible to have a more specific selector to avoid using `!important`?

Reply:
I think not - it conflicts with main styles that manage to override this.